BitwardenShared/UI/Platform/Application/AppProcessorTests.swift

```diff
@@ -200,6 +200,46 @@ class AppProcessorTests: BitwardenTestCase { // swiftlint:disable:this type_body
         XCTAssertIdentical(syncService.delegate, subject)
     }
 
+    /// `handleAppLinks(URL)` navigates the user based on the input URL.
+    func test_init_handleAppLinks() {
```

Comment:
🎨 There should be tests for the different regions and also for when `fromEmail` can't be casted to `Bool` thus defaults to `true`.